command/helper/config.go

```diff
@@ -298,6 +305,11 @@ func (c *Config) mergeConfigWith(otherConfig *Config) error {
 		c.RestoreFile = otherConfig.RestoreFile
 	}
 
+	// if block time not default, set to new value
+	if otherConfig.BlockTime != 2000 {
```

Comment:
Here as well

Reply:
Defatult value moved to const 631515404cf370d1fc015938ea35da52ff9dbaaf